src/components/z-tag/styles.css

```
@@ -0,0 +1,53 @@
+:host {
+  --z-icon-width: 15px;
+  --z-icon-height: 15px;
```

io da abstract vedo 14px

```suggestion
  --z-icon-width: 14px;
  --z-icon-height: 14px;
```